### PR TITLE
[Docs] Clean up README and fix logo

### PR DESCRIPTION
**Pull request by tuxxy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,15 @@
...
 
-.. image:: https://circleci.com/gh/nucypher/nucypher/tree/master.svg?style=svg
-    :target: https://circleci.com/gh/nucypher/nucypher/tree/master
+----
 
+![](https://img.shields.io/pypi/wheel/nucypher.svg)
+![](https://img.shields.io/pypi/pyversions/nucypher.svg)
+![](https://coveralls.io/repos/github/nucypher/nucypher/badge.svg?branch=master)
+![](https://circleci.com/gh/nucypher/nucypher/tree/master.svg?style=svg)
+![](/docs/source/.static/img/GPLv3.png)
```
Comment (KPrasch) [inline README.md]: How about  ->

**Comment by KPrasch** [comment]
Can you squash some of these commits to ~3?

File: README.md (hunk 1 above)
Comment (tuxxy) [reply README.md]: Why does it say missing? I just use the GPLv3 logo. It's not like we need to show the status of our license lol

Comment (KPrasch) [reply README.md]: It says missing because the package it not yet on PyPI - No - but I prefer the uniformity pf the badges

Comment (KPrasch) [reply README.md]: Plus - that logo looks like a 90's power ranger

Comment (tuxxy) [reply README.md]: Let's do it incrementally, once we get the package on PyPI then let's switch to that badge.

Comment (tuxxy) [reply README.md]: I don't want it to say "license: missing" when it really shouldn't.

Comment (KPrasch) [reply README.md]: Well - we already do this in several places in the docs (the other badges for example), and I anticipate the period of time before publication to be small - I think a follow-up PR on this basis is overkill.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,21 @@
...
 
+![](https://img.shields.io/pypi/wheel/nucypher.svg)
+![](https://img.shields.io/pypi/pyversions/nucypher.svg)
+![](https://coveralls.io/repos/github/nucypher/nucypher/badge.svg?branch=master)
+![](https://circleci.com/gh/nucypher/nucypher/tree/master.svg?style=svg)
+![](/docs/source/.static/img/GPLv3.png)
```
Comment (KPrasch) [inline README.md]: These badges are included twice

**Pull request merged**